web/components/add-funds-modal.tsx

```diff
@@ -84,7 +84,7 @@ function BuyManaTab(props: { onClose: () => void }) {
   return (
     <>
       <div className="mt-6 mb-4">
-        Buy mana (M$) to trade in your favorite markets.
+        Buy mana (Ṁ) to trade in your favorite markets.
```

Comment:
Rather than changing it to the new symbol, you should use the `ENV_CONFIG.moneyMoniker` constant, so it's easier to change in the future.

Ditto for all the other references.